app/controllers/student_task_controller.rb

```diff
@@ -97,5 +103,42 @@ def publishing_rights_update
     end
   end
 
+  def email_reviewers; end
+
+  def send_email
+    subject = params['send_email']['subject']
+    body = params['send_email']['email_body']
+    participant_id = params['participant_id']
+    assignment_id = params['assignment_id']
+    @participant = AssignmentParticipant.find_by(participant_id)
```

Comment:
```suggestion
    @participant = AssignmentParticipant.find_by(id: participant_id)
```
Think this might work?